src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt

```diff
@@ -32,26 +32,22 @@
 ############################################################################
 
 if(NOT GENERATE_RTPS_BRIDGE MATCHES "off")
+
     find_program(FASTRTPSGEN fastrtpsgen
                  PATHS $ENV{FASTRTPSGEN_DIR})
     if(NOT FASTRTPSGEN)
-        message(WARNING "WARNING: Unable to find fastrtpsgen")
-        set(GENERATE_RTPS_BRIDGE off)
+        message(WARNING "WARNING: Unable to find fastrtpsgen. Building PX4 without RTPS bridge support")
```

Comment:
This change breaks the required behavior that if the build system does not have fastrtpsgen, it will not try to run the fastrtpsgen command.